tests/e2e/e2e_test.go

```diff
@@ -44,6 +44,23 @@ func (s *IntegrationTestSuite) TestIBCTokenTransfer() {
 	chainB.SendIBC(chainA, chainA.NodeConfigs[0].PublicAddress, initialization.StakeToken)
 }
 
+func (s *IntegrationTestSuite) TestIBCTokenTransferRateLimiting() {
+	// TODO: Add E2E tests for this
+	if s.skipIBC {
+		s.T().Skip("Skipping IBC tests")
+	}
+	chainA := s.configurer.GetChainConfig(0)
+	chainB := s.configurer.GetChainConfig(1)
+
+	//node, err := chainA.GetDefaultNode()
+	//s.NoError(err)
+	// This doesn't work. Why?
+	//node.StoreWasmCode("rate_limiter.wasm", initialization.ValidatorWalletName)
```

Comment:
I'd guess that its an issue with the rate_limiter.wasm filepath. Golang measures filepaths relative to where the binary is being executed

Reply:
Oh also wouldn't work in this PR since IBC rate limit isn't wired up

Reply:
yeah, I'm getting these to run on the other branch. Slowly moving forward cause I still have to run the full suite every time to get e2e to work 😂